tests/integration/api/test_project.py

```diff
@@ -481,6 +481,8 @@ def assert_project_data_details(data: Dict[str, Any], project: models.Project):
             "credential_type": credential.credential_type,
             "username": credential.username,
             "vault_identifier": None,
+            "scm_ssh_key": None,
+            "scm_ssh_key_password": None,
```

Comment:
@jshimkus-rh Should we use passphrase instead of password since that is the well known term for private keys